src/sfizz/Region.cpp

```diff
@@ -1692,6 +1692,10 @@ bool sfz::Region::processGenericCc(const Opcode& opcode, OpcodeSpec<float> spec,
         auto it = std::find_if(connections.begin(), connections.end(),
             [ccNumber, &target](const Connection& x) -> bool
             {
+                if ((ccNumber > 130 && ccNumber < 138) || ccNumber == 140 || ccNumber == 141)
```

Comment:
If we keep it like this maybe it would make sense to add an `isPerVoiceController` function to the extended CC list to replace both this check and the switch below in this method. However we can plan to do that after.